### PR TITLE
Adding Snake Game

### LINKED ISSUE #5: [New game]: Snake Game
### 🎮 Game Request

It will be a simple game where the snake can move around in any direction in the square and eat the fruit and it will become longer and longer.
This game will use the tech stack HTML, CSS and JavaScript.

### Point down the features

Snake will be of variable length
Fruit will appear randomly one after another
Easy UI to play

### Select program in which you are contributing

GSSoC23

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by lmalkam**
## PR Description 📜

I have added the snake game on the website.
Fixes #5 

<hr>
 
## Mark the task you have completed ✅
issue #5 is solved

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [ ] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸
image

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
@lmalkam Please make sure you mark the tasks that you are done with 😃 
Please mark the tasks you are completed with 🤝 !!

**Comment by lmalkam** [comment]
@kunjgit I marked all the tasks and confirmed it!!

**Review by kunjgit — COMMENTED** [review]
fix the image and you are good to go ! 🚀

File: assets/images/SnakeGame.png
[the remark this answers is not in the document]
Comment (lmalkam) [reply assets/images/SnakeGame.png:1]: Please check and review !!
And if any improvements needed please comment!  @kunjgit

**Review by kunjgit — APPROVED** [review]

**Comment by kunjgit** [comment]
Thank you  @lmalkam , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**